packages/performance-tests/test/TestContext.ts

```diff
@@ -8,9 +8,33 @@ import { Logger } from "@itwin/core-bentley";
 import { BriefcaseIdValue, IModelVersion } from "@itwin/core-common";
 import { AccessTokenAdapter, BackendIModelsAccess } from "@itwin/imodels-access-backend";
 import assert from "assert";
+import { generateTestIModel, IModelParams } from "./iModelUtils";
 
 const loggerCategory = "TestContext";
 
+export interface ReporterInfo {
```

Comment:
Is this a good place for these helper interfaces. Or maybe move them into a reporterUtils file? Don't know if its better to add to this file, or create yet another helper file. Also i've been considering moving those util files into a `RegTestUtils` dir or something

Reply:
done